packages/rsocket-core/src/Fragmenter.ts

```diff
@@ -57,7 +57,7 @@ export function* fragment(
 
     if (metadataLength === 0) {
       remaining -= Lengths.METADATA;
-      metadata = Buffer.from([]);
+      metadata = Buffer.allocUnsafe(0);
```

Comment:
What is the benefit of moving to `allocUnsafe` here? We should be mindful that this does not clear out the buffer, so it would be possible to leak any data that was previously in that memory.

Reply:
I believe that `allocUnsafe` do stuff faster and since we are rewriting content in any case I consider that this "should" be safe. But I'm not 100% sure

Reply:
Will think more on this before we ship `1.0`, but should be OK for now.